README.md

```diff
@@ -1,9 +1,11 @@
 # obs_ctio0m9
 A repository to hold the description for ctio0m9
 
+*Warning:* This package is deprecated, and will be removed from the Rubin Observatory Science Pipelines distribution after release 21.0.0.
+
 The data taking at this site was not fully automated, and so some header keywords are known to be unreliable.
 Some of these are standardised during ingest by the obs_package (e.g. normalisation of filter naming,
 ensuring the DATE-OBS keyword is ISO8601 compliant etc). However, some things are, by necessity, done
 in a pre-ingest script (one needs to visually inspect flats to ensure they are not actually dispersed images)
 
-For the data up-to and including March 2017, this can be found in the /pre_ingest/sanitize.ipynb
\ No newline at end of file
+For the data up-to and including March 2017, this can be found in the /pre_ingest/sanitize.ipynb
```

Comment:
I can't see any difference on this line so I'm not actually sure why it's showing in the changes, but obviously looks fine, given that it's apparently unchanged.

Reply:
The line now ends with a new line character (which GitHub isn't displaying — I thought it had some symbol for that, but maybe not). This is almost always the right thing to do for a text file (https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).